test/integrationTests/virtualDocumentTracker.integration.test.ts

```diff
@@ -15,7 +15,7 @@ chai.use(require('chai-arrays'));
 chai.use(require('chai-fs'));
 
 suite(`Virtual Document Tracking ${testAssetWorkspace.description}`, function () {
-    let virtualScheme: string = "virtual";
+    let virtualScheme: string = "virtualCSharp-test";
```

Comment:
We should probably add a negative test, ie documents with a scheme that doesn't start with virtualCsharp- aren't tracked.

Reply:
Gonna have to find a better way to test the virtual documents. The server still understands requests that are specifically invoked on virtual documents but it treats them as misc docs therefore they don't impact anything surrounding them.